README.md

```diff
@@ -38,22 +37,22 @@ This 'Quick Start' template allows the configuration of multiple integrations fr
 
 ### Required Inputs
 
-- `HoneycombAPIKey`: : Your Honeycomb Team's API Key.
+* `HoneycombAPIKey`: : Your Honeycomb Team's API Key.
 
-*Note*: All other Parameters are optional but without providing any additional parameters all that will be created is a S3 Bucket.
+*Note*: All other Parameters are optional, but if no additional parameters are provided, only a S3 Bucket will be created.
 
 ## Cloudwatch Logs
 
-This template supports integrating with up to 6 Cloudwatch Log Groups and shipping them to a Honeycomb dataset.
+This template supports integrating with up to six Cloudwatch Log Groups and shipping them to a Honeycomb dataset.
 
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https://honeycomb-builds.s3.amazonaws.com/cloudformation-templates/latest/cloudwatch-logs.yml&stackName=Honeycomb-Logs" target="_blank"><img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png" alt="Launch Stack" /></a>
 
 ### Required Inputs
 
-- `HoneycombAPIKey`: Your Honeycomb Team's API Key.
-- `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
-- `LogGroupName`: A CloudWatch Log Group name. Additional Log Groups can be added with the `LogGroupNameX` parameters.
-- `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
+* `HoneycombAPIKey`: Your Honeycomb Team's API Key.
+* `HoneycombDataset`: The target Honeycomb dataset for the Stream to publish to.
+* `LogGroupName`: A CloudWatch Log Group name. Additional Log Groups can be added with the `LogGroupNameX` parameters.
+* `S3FailureBucketArn`: The ARN of the S3 Bucket that will store any logs that failed to be sent to Honeycomb.
```

Comment:
Also what is ARN? It's not defined on this page anywhere.

Reply:
Hmm.. tricky. We don't define S3 or CloudWatch either and all are AWS specific jargon.

The AWS definition: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html

Reply:
Well, S3 and CloudWatch are more mainstream as product name while ARN as a term is new to me.

...we _can_ define those terms, too, at least once though...😈

Reply:
hmm I feel like lets wait and iterate on how much we need to explain AWS domain? I feel like knowing what it stands for might not help someone who does not know what it is and how to get it - they might need to go look it up in AWS docs anyway?

Reply:
Okay. Will hold off then.